### PR TITLE
Update RELEASE.md with pinning xla and builder PRs

### PR DESCRIPTION
**Pull request by atalman**
Provide example PRs necessary for pinning xla and builder repos for release

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -95,6 +95,8 @@ them:
 * Update backwards compatibility tests to use RC binaries instead of nightlies
   * Example: https://github.com/pytorch/pytorch/pull/77983 and https://github.com/pytorch/pytorch/pull/77986
 * A release branches should also be created in [`pytorch/xla`](https://github.com/pytorch/xla) and [`pytorch/builder`](https://github.com/pytorch/builder) repos and pinned in `pytorch/pytorch`
+  * Example: https://github.com/pytorch/pytorch/pull/86290 and https://github.com/pytorch/pytorch/pull/90506 
+
```
Comment (atalman) [inline RELEASE.md:99]:
```suggestion
```

**Review by malfet — APPROVED** [review]
Sure, but it'll probably be good to merge those two into one PR

Also, as a BE fix, why can't we use the pin trough-out the codebase?

**Review by kit1980 — APPROVED** [review]

File: RELEASE.md [hunk 2]
```diff
@@ -95,6 +95,7 @@ them:
 * Update backwards compatibility tests to use RC binaries instead of nightlies
   * Example: https://github.com/pytorch/pytorch/pull/77983 and https://github.com/pytorch/pytorch/pull/77986
 * A release branches should also be created in [`pytorch/xla`](https://github.com/pytorch/xla) and [`pytorch/builder`](https://github.com/pytorch/builder) repos and pinned in `pytorch/pytorch`
+  * Example: https://github.com/pytorch/pytorch/pull/86290 and https://github.com/pytorch/pytorch/pull/90506
```
Comment (atalman) [inline RELEASE.md:98]:
```suggestion
  * Example: https://github.com/pytorch/pytorch/pull/86290 and https://github.com/pytorch/pytorch/pull/90506
```

**Pull request closed without merging**